src/main/java/ch/dvbern/kibon/exchange/api/common/platzbestaetigung/BetreuungAnfrageDTO.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2020 DV Bern AG, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.dvbern.kibon.exchange.api.common.platzbestaetigung;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.StringJoiner;
+
+import javax.annotation.Nonnull;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import ch.dvbern.kibon.exchange.api.common.shared.GesuchstellerDTO;
+import ch.dvbern.kibon.exchange.api.common.shared.KindDTO;
+
+public class BetreuungAnfrageDTO implements Serializable {
```

Comment:
I just realised, that BetreuungsAngebot should already be set by Gesuchsteller. There could be use cases where it's useful information for an application. I would add another field `betreuungsArt` (as in `VerfuegungDTO`) for it.